packages/tracing/src/browser/browsertracing.ts

```diff
@@ -96,13 +136,40 @@ export class BrowserTracing implements Integration {
   public setupOnce(_: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
     this._getCurrentHub = getCurrentHub;
 
-    const { routingInstrumentation, startTransactionOnLocationChange, startTransactionOnPageLoad } = this.options;
+    if (this._emitOptionsWarning) {
```

Comment:
`_emitOptionsWarning` is based on `tracingOrigins` only, which we have access to here. Not sure if it's necessary to store it in the class.